### ISSUE TITLE
Are you expected to have to escape backslashes in text fields?

### ISSUE THREAD

**Issue by lemongrabber** [ConfigHubPub/ConfigHubPlatform#11]
We noticed when inserting some settings into confighub, that strings with backslashes in them (windows file paths for example) need to have their backslashes escaped in order to have them show up properly in the preview GUI. We were just wondering what the expected behavior of this should be?

**Comment by edoarsla** [comment]
This is resolved as of v1.3.6.

If you have a key with a value like:  `"C:\WINDOWS\foo\bar.tmp"`
Preview will show it the same as entered, and resolved JSON will contain something like:

```
{
  "generatedOn": "05/30/2017 18:11:15",
  "account": "test",
  "repo": "test",
  "context": "Foo;Bar",
  "files": {
    "server/conf/tomee.xml": {
      "content": "JdbcUrl jdbc:mysql://C:\\WINDOWS\\foo\\bar.tmp",
      "content-type": "application/xml"
    }
  },
  "properties": {
    "db.api.host": {
      "val": "C:\\WINDOWS\\foo\\bar.tmp"
    }
  }
}
```

**Comment by lemongrabber** [comment]
Awesome, thank you!
